### DISCUSSION TITLE
aws-ebs-csi-driver in cluster install

### DISCUSSION THREAD

**Discussion by ramialt** [milvus-io/milvus#25218] [Q&A and General discussion]
Milvus cluster install docs include aws-ebs-csi-driver in the milvus_cluster.yaml file with the comment "Optional" like so:
```
addons:
- name: aws-ebs-csi-driver
  version: v1.13.0-eksbuild.1 # optional
``` 
What is optional here? The addon itself, or the specific version?

I am also curious to learn how this driver is used in Milvus.

Thanks

**Comment by yhmo** [comment]
I guess it is about the specific version.

**Reply by ramialt** [reply]
If a version is not specified, will it just find the latest somehow? Couldn't find anything about this in docs

**Comment by LoveEachDay** [comment]
`aws-ebs-csi-driver` is used to automatically provision a volume for milvus using storageclass like `gp2`.
